IHP/ModelSupport.hs

```diff
@@ -594,10 +589,10 @@ deleteRecord model = do
 -- >>> delete projectId
 -- DELETE FROM projects WHERE id = '..'
 --
-deleteRecordById :: forall model id. (?modelContext :: ModelContext, Show id, Table model, HasField "id" model id, ToField id) => id -> IO ()
+deleteRecordById :: forall record id. (?modelContext :: ModelContext, Show id, Table record, ToField id) => id -> IO ()
 deleteRecordById id = do
```

Comment:
This now compiles. However, I tried to cut down a bit more on code and have 

```haskell
deleteRecordById :: forall record id. (?modelContext :: ModelContext, Show id, Table record, ToField id) => id -> IO ()
deleteRecordById id = deleteRecordByIds [ids]
```

But maybe that's not possible, since we don't use the `record` - but can't remove it?

![image](https://user-images.githubusercontent.com/125707/150841749-ff7c354c-083d-47af-b87f-64e2c59f930a.png)

Reply:
@mpscholten do you think the above is possible?